CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.0.0] - TBD
+### Added
+- Add basic Time To Live (TTL) for all tables.
+- DB migration for creating new `housekeeping_metadata` table.
+### Changed
+- Renamed `beekeeper-path-scheduler-apiary` module to `beekeeper-scheduler-apiary`.
+- Renamed `beekeeper-path-apiary` module to `beekeeper-apiary`.
+- Renamed `beekeeper-assembly-path-scheduler-apiary` module to `beekeeper-assembly-scheduler-apiary`.
+- DB migration to rename `path_status` column to `housekeeping_status` in the `path` table.
+- DB migration to rename `path` table to `housekeeping_path` table.
+
```

Comment:
please add:
```- This is a breaking change and an existing database needs to be migrated please see the README.md#MigratingFrom2.1.0-3.0.0 for instructions.
```
And write up a little section explaining what needs to be done.
 E.g. "run the following script(s) which you can find here:....." etc..

This needs to be a bit more clear our future self will thank us :)